pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1470,7 +1470,15 @@ func CheckNodeUnschedulablePredicate(pod *v1.Pod, meta algorithm.PredicateMetada
 		return false, []algorithm.PredicateFailureReason{ErrNodeUnknownCondition}, nil
 	}
 
-	if nodeInfo.Node().Spec.Unschedulable {
+	// If pod tolerate unschedulable taint, it's also tolerate node.Spec.Unschedulable.
+	unschedulable := !v1helper.TolerationsTolerateTaint(pod.Spec.Tolerations, &v1.Taint{
+		Key:    algorithm.TaintNodeUnschedulable,
+		Effect: v1.TaintEffectNoSchedule,
+	})
+	// TODO (k82cn): deprecates `node.Spec.Unschedulable` in 1.13.
```

Comment:
it's unclear what is being deprecated here, or what action should be taken in 1.13

Reply:
Update related doc to depreciate it.